phpunit.xml

```diff
@@ -15,10 +15,17 @@
     </testsuites>
 
     <logging>
-        <log type="coverage-html" target="tests/coverage/"/>
+        <log type="coverage-clover" target="build/logs/clover.xml"/>
```

Comment:
I'm pretty sure it will :) By adding these lines in phpunit.xml, you make the configuration static. This config can also be defined at runtime using similar option:

`phpunit --coverage-clover build/logs/clover.xml` is the same as `<log type="coverage-clover" target="build/logs/clover.xml"/>`